learning/courses/quantum-diagonalization-algorithms/vqe.ipynb

```diff
@@ -694,64 +694,61 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 50,
+   "execution_count": 13,
    "id": "8e0d33b8-7f9c-428d-a76d-d832747b8430",
    "metadata": {},
    "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/home/porter284/.pyenv/versions/3.11.12/lib/python3.11/site-packages/qiskit_ibm_runtime/qiskit_runtime_service.py:1129: UserWarning: IBM Quantum Platform is moving, and this version will be sunset on July 1. To get started on the new platform, read the migration guide at https://docs.quantum.ibm.com/migration-guides/classic-iqp-to-cloud-iqp\n",
```

Comment:
UserWarning here @christopherporter1

Reply:
Resolved in https://github.com/Qiskit/documentation/pull/3463/commits/8a5a65fe590b6d2273897d15ba7a47c87fd76fc6